doc/user/sql-grammar/sql-grammar.bnf

```diff
@@ -312,8 +320,7 @@ sink_kafka_connector ::=
     'KAFKA BROKER' host 'TOPIC' topic-prefix
     ('KEY' '(' key_column ( ',' key_column )* ')')?
     ('CONSISTENCY' 'TOPIC' topic ('CONSISTENCY' 'FORMAT' 'AVRO USING' 'CONFLUENT SCHEMA REGISTRY' url)?)?
-    with_options?
-    'FORMAT' ('AVRO USING' 'CONFLUENT SCHEMA REGISTRY' url | 'JSON')
+    ( 'WITH' '(' (option_name ('=' option_value)?) ( ',' (option_name ('=' option_value)?) )* ')' )?
```

Comment:
Is it is now, we list the `WITH` block twice in the railroad diagrams. Once in the toplevel diagram and then again at the end of `kafka_connector`. Maybe we should remove it here?

Reply:
I don't think we can do much about the length of "CONSISTENCY TOPIC" right now (I am going to spend this Friday looking at a fix, I think), but maybe we can break the second set of "WITH" options into its own diagram so at least it will be flush left instead of crossing over into our right navigation.

Reply:
So we actually do have two `WITH` blocks, which caused [this bug](https://github.com/MaterializeInc/materialize/issues/8201)! I'm working on that now, and I can wait until that fix is finished to update the diagrams, if that seems better. (Otherwise, I'll just open another PR.)

Reply:
Oh boy! :sweat_smile: Are we planning to get rid of the second `WITH` block?